### PR TITLE
adding spopt

### PR DESCRIPTION
**Pull request by jGaboardi**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jGaboardi** [comment]
@knaaptime and I are not sure why the `staged-recipe` build is [failing](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=265211&view=logs&jobId=6f142865-96c3-535c-b7ea-873d86b887bd&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823) here. The [`spopt` CI is passing](https://github.com/pysal/spopt/actions/runs/489221363) without issue. @ocefpaf Any thoughts or advice on this would be greatly appreciated.

**Comment by jGaboardi** [comment]
pysal/spopt#118

**Comment by jGaboardi** [comment]
It seems that there is [a problem](https://github.com/pysal/spopt/issues/118#issuecomment-761828515) with our build on PyPI.

**Comment by jGaboardi** [comment]
> GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.

@knaaptime Can you post a comment here?

**Comment by knaaptime** [comment]
I'm happy to help maintain

File: recipes/spopt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    
+test:
+  imports:
+    - {{ name }}
+  # --> comment from @ocefpaf <-- see conda-forge/libpysal-feedstock#8
+  # pip check is not smart enough to know that an import name is not the package name :-(
+  # commands:
+  #  - pip check
+  # requires:
+  #  - pip
```
[the remark this answers is not in the document]
Comment (jGaboardi) [reply recipes/spopt/meta.yaml:41]: This was a relic from the [`spaghetti-feedstock`](https://github.com/conda-forge/spaghetti-feedstock/blob/master/recipe/meta.yaml#L34), which in turn was based on the [`libpysal-feedstock`](https://github.com/conda-forge/libpysal-feedstock/blob/25530dcff04e0a668157959abf0b25b1a96f3f9a/recipe/meta.yaml#L43).

**Pull request merged**